packages/core/src/api.js

```diff
@@ -91,14 +95,20 @@ export function createPercyServer(percy, port) {
     })
   // post one or more snapshots, optionally async
     .route('post', '/percy/snapshot', async (req, res) => {
-      let snapshot = percy.snapshot(req.body);
+      let snapshot = req.body.enableSync
+        ? await promiseWrapper(percy.snapshot, req.body)
+        : percy.snapshot(req.body);
```

Comment:
We dont need to call in 2 ways, we can just call once and choose to await depending on sync. It makes it easier to maintain when more params are added